components/wpt-results.html

```diff
@@ -46,59 +120,170 @@
 
       static get properties() {
         return {
+          query: {
+            type: String,
+            value: '',
+            observer: '_queryChanged'
+          },
           specs: {
             type: Array,
-            observer: '_specsChanged',
             value: []
-          }
+          },
+          testRuns: {
+            type: Array,
+          },
         };
       }
 
-      connectedCallback() {
+      async connectedCallback() {
         super.connectedCallback();
 
-        this._fetchResults()
+        const testFileResults = await Promise.all(this.testRuns.map(testRun => {
+          return this._fetchResults(testRun.results_url)
+        }))
+
+        let specs = []
+
+        testFileResults.forEach(result => {
+          const testFiles = result.testFiles
+          const resultsURL = result.resultsURL
```

Comment:
You mean using the GitHub issue tracker?

It's probably simpler to keep all issues in one place, so we should decide on either GitHub or Monorail. I'm kind of in favor of Monorail since I'm more used to it :-P